grin/src/Reducer/LLVM/CodeGen.hs

```diff
@@ -434,7 +434,7 @@ codeGenCase opVal alts bindingGen = do
         (Alt DefaultPat _, _) -> True
         _ -> False
       (defaultAlts, normalAlts) = List.partition isDefault alts
-  when (length defaultAlts > 1) $ fail "multiple default patterns"
```

Comment:
This was complaining that `Identity` didn't implement `MonadFail`.  
I think that means it was calling `error` before this change anyway.